prow/plugins/help/help.go

```diff
@@ -30,19 +30,32 @@ import (
 const pluginName = "help"
 
 var (
-	helpLabel         = "help wanted"
-	helpRe            = regexp.MustCompile(`(?mi)^/help\s*$`)
-	helpRemoveRe      = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
-	helpGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
-	helpMsgPruneMatch = "This request has been marked as needing help from a contributor."
-	helpMsg           = `
+	helpLabel                  = "help wanted"
+	goodFirstIssueLabel        = "good first issue"
+	helpRe                     = regexp.MustCompile(`(?mi)^/help\s*$`)
+	helpRemoveRe               = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
+	helpGoodFirstIssueRe       = regexp.MustCompile(`(?mi)^/help good-first-issue\s*$`)
+	helpGoodFirstIssueRemoveRe = regexp.MustCompile(`(?mi)^/help remove-good-first-issue\s*$`)
```

Comment:
I'm not sure if this is how the command should be.

Reply:
Could probably omit the `help` bit

Reply:
`/remove-help good-first-issue`?

Reply:
what about just `/good-first-issue` and `/remove-good-first-issue`, but still be integrated in the help plugin? It's more intuitive this way.

Reply:
I really like `/good-first-issue`, seems more consistent with other commands.

Reply:
I don't mind it, but the only concern in my mind is it doesn't align with good first issue being a "subset" label of help wanted.

Reply:
> the only concern in my mind is it doesn't align with good first issue being a "subset" label of help wanted.

yeah, if it's intuitive one way, it's not intuitive the other way :/

Reply:
updated to `/good-first-issue` and `/remove-good-first-issue`